Gemfile.base

```diff
@@ -69,6 +69,7 @@ gem 'audited'
 gem 'acts_as_list', '~> 0.9.17'
 gem 'braintree', '~> 2.89'
 gem 'cancancan', '~> 2.3.0'
+gem 'draper-cancancan'
```

Comment:
Because otherwise, any `can?` with the object decorated does not work as we wish because the decorated object is not the same as the model object.

Reply:
Or we can add the method ourselves without the gem. It's only one method that needs to be patched (CanCan::Ability#can?) after all, and whose implementation can be improved BTW.

Reply:
Fair enough 👍 💪